hw/top_englishbreakfast/data/clocks.xdc

```diff
@@ -0,0 +1,28 @@
+## Copyright lowRISC contributors.
```

Comment:
this is probably another PR, but i feel like we should now separate these xdc's into a "core" part and "customizable" part... maybe it won't be that easy... and it'll be a lot of iterations...

Reply:
It's hard to say how much we would gain from doing this. What could maybe be done is to extract the top name from the design. I did that for another Vivado script but don't know if this would work for a constraints file. We could then replace the file by a symlink, too. I will look into that.

Reply:
ah no worries, i think as you say, the bang for buck here is much smaller compared to stuff like the others. It was just a thought.  I have a feeling for similar projects the constraints really won't change a ton.

Reply:
I have now replaced that file by a symlink as well. I took me a while to figure out that xdc files support wildcard matching. So we don't actually need to specify the top_name in here.